### DISCUSSION TITLE
Node Status : NOT Synced

### DISCUSSION THREAD

**Discussion by djtellado** [Chia-Network/chia-blockchain#4432] [Q&A]
from yesterday I loose the synchro for the Node. I change the version to 1.1.5 but still the same. Connection Status is Connected. And I can plotting, but I can not farm.

Any advice?
<img width="1056" alt="Captura de pantalla 2021-05-10 a las 14 43 26" src="https://user-images.githubusercontent.com/83960117/117660984-162e1500-b19e-11eb-944a-dcf7a6a48b3b.png">

**Comment by fjbotto** [comment]
I'm having also the same issue on Ubuntu 20

**Reply by djtellado** [reply]
Mine is already resolved, just leave it connected ....it will come back to normal. I have Mac OS but I think is a problem of saturation due the error yesterday
